### PR TITLE
Add open new tab in resource view(connects #2073)

### PR DESCRIPTION
**Pull request by jiajunc**
Add "open new tab"  button to resource view.

### CODE REVIEW

File: src/app/resources/view-resources/resources-view.component.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
     </button>
     <span class="toolbar-fill"></span>
     <ng-container *ngIf="!parent">
+      <a mat-raised-button [href]="resourceSrc" target="_blank" color="accent">Open in new tab</a>
```
Comment (yuviii) [inline src/app/resources/view-resources/resources-view.component.html]: add i18n here.
      <a mat-raised-button [href]="resourceSrc" target="_blank" color="accent" i18n>Open in new tab</a>

File: src/app/users/users-profile/users-profile.component.html [hunk 2]
```diff
@@ -9,8 +9,8 @@
     <span>{{userDetail.name}}</span>
     <mat-icon class="margin-lr-5" *ngIf="userDetail.gender" svgIcon="{{userDetail.gender}}"></mat-icon>
     <span class="toolbar-fill"></span>
-    <button mat-button class="margin-lr-3" *ngIf="user.name === urlName || user.isUserAdmin" i18n [routerLink]="['../../update/', urlName]"><mat-icon>mode_edit</mat-icon><span>Edit Profile</span></button>
-    <a mat-button class="margin-lr-3" *ngIf="user.name === urlName" i18n [planetChangePassword]="userDetail">Change password</a>
+    <button mat-raised-button color="accent" class="margin-lr-3" *ngIf="user.name === urlName || user.isUserAdmin" i18n [routerLink]="['../../update/', urlName]"><mat-icon>mode_edit</mat-icon><span>Edit Profile</span></button>
```
Comment (lmmrssa) [inline src/app/users/users-profile/users-profile.component.html]: these changes are not related to this PR looks like you created branch from another branch. Please make sure you always create branch from **master** only
